docs/pages/kubernetes-access/discovery/includes/troubleshooting.mdx

```diff
@@ -0,0 +1,26 @@
+### Discovery Service
+
+To check if the Discovery Service is working correctly, you can check if any Kubernetes 
+clusters have been discovered. To do this, you can use the `tctl get kube_cluster` 
+command and inspect if the expected clusters have already been imported into Teleport.
+
+If some clusters do not appear in the list, check if the filtering labels match 
+the missing cluster tags or look into the service logs for permission errors.
+
+### Kubernetes Service
+
+If the `tctl get kube_cluster` command returns the discovered clusters, but the 
+`tsh kube ls` command does not include them, check that you have set the 
```

Comment:
This also might be a Teleport RBAC issue (your role doesn't let you see them), I would mention this too and give an example of what a role should look like.